ethcore/Cargo.toml

```diff
@@ -32,6 +32,7 @@ ethcore-logger = { path = "../logger" }
 ethcore-stratum = { path = "../stratum" }
 ethcore-util = { path = "../util" }
 ethcore-bigint = { path = "../util/bigint" }
+memory_cache = { path = "../util/memory_cache" }
```

Comment:
see above lines: naming convention for crates is with dashes, not hyphens.

Reply:
actually, we are very inconstant with it. e.g. `rlp_derive`, `semantic_version`, `rpc_cli`, `secret_store` and so on...

Reply:
Fixed